src/main/kotlin/me/serce/solidity/ide/inspections/fixes/ImportFileAction.kt

```diff
@@ -94,10 +95,26 @@ class ImportFileAction(
   }
 
   companion object {
+    fun isImportedAlready(file: PsiFile, to: PsiFile): Boolean {
+      return if (file == to) {
+        true
+      } else {
+        RecursionManager.doPreventingRecursion(file, true) {
+          file.children
+            .filterIsInstance<SolImportDirective>()
+            .mapNotNull { it.importPath?.reference?.resolve()?.containingFile }
+            .any {
+              isImportedAlready(it, to)
+            }
+        }!!
```

Comment:
`RecursionManager.doPreventingRecursion` might return `null` if it encounters a recursion, see

https://github.com/JetBrains/intellij-community/blob/master/platform/util/src/com/intellij/openapi/util/RecursionGuard.java#L36, 

so I think we should provide a default value here, `true` seems to be the most appropriate for this case

Reply:
@SerCeMan, I added null-handling. But I disagree about default `true`. False should be returned in this case.
Added test for this situation

Reply:
Oh, I see. So, we still want to perform the action even though there is a recursion.